turbo.json

```diff
@@ -12,6 +12,7 @@
     },
     "test": {},
     "dev": {
+      "dependsOn": ["^build"],
```

Comment:
Is this true? doesn't it not need to run the build step for the extension? (definitely not for web, right?)

Reply:
oh wait nvm, the `^` thing